### ISSUE TITLE
AisMessage19 has duplicate property names AisStaticCommon

### ISSUE THREAD

**Issue by zig158** [dma-ais/AisLib#22]
AisMessage19 has duplicate property names/is hiding several protected properties of its super class AisStaticCommon. 
I consider this a bug since
1) it breaks json serialization libraries such as Gson for no good reason.
2) it is bad practice.

**Comment by mablae** [comment]
Sounds realistic. Maybe @tbsalling can say sth? He has written some patches regarding 
Serializable bot long time ago.

**Comment by tbsalling** [comment]
@zig158 could you be more specific? - are you referring to these?

```
    private int dimBow; // 9 bits
    private int dimStern; // 9 bits
    private int dimPort; // 6 bits
    private int dimStarboard; // 6 bits
```

**Comment by tbsalling** [comment]
I'm on it :-) To be more specific, I suppose we're talking about:

```
name
shipType
dimStern
dimBow
dimStarbord
dimPort
```

**Comment by tbsalling** [comment]
Resolved by commit 35224dd170e848caf2abc3bb725dbf2ecf8ac441. Comments welcome.

**Comment by mablae** [comment]
Time to try gson again :-)  I was running into the same problem two years ago, 
and ended up serializing an handcrafted Dictionary instead. 

Thank you @tbsalling
